lxc/config.go

```diff
@@ -631,9 +631,9 @@ func (c *cmdConfigSet) Run(cmd *cobra.Command, args []string) error {
 				}
 
 				return op.Wait()
-			} else {
-				return fmt.Errorf(i18n.G("The is no config key to set on an instance snapshot."))
 			}
+
+			return fmt.Errorf(i18n.G("The is no config key to set on an instance snapshot."))
```

Comment:
The error message itself doesn't make sense, should this be "There" rather than "The"?

Reply:
Good catch, tbh I haven't really looked at the message itself.

Reply:
fixed and updated the translations.